src/styles/login.css

```diff
@@ -0,0 +1,296 @@
+@import url('https://fonts.googleapis.com/css2?family=Gothic+A1&family=Montserrat&family=Roboto&display=swap');
+*{
+    padding:0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5554130)